hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/RealtimeUnmergedRecordReader.java

```diff
@@ -74,7 +78,7 @@ public RealtimeUnmergedRecordReader(RealtimeSplit split, JobConf job,
     this.parquetRecordsIterator = new RecordReaderValueIterator<>(this.parquetReader);
     this.executor = new BoundedInMemoryExecutor<>(
         HoodieRealtimeRecordReaderUtils.getMaxCompactionMemoryInBytes(jobConf), getParallelProducers(),
-        Option.empty(), x -> x, new DefaultSizeEstimator<>());
+        Option.empty(), Function.identity(), new DefaultSizeEstimator<>(), Functions.noop());
```

Comment:
we already have a overloaded constructor which sets Functions.noop() as last arg. we don't need to set explicitly here.

Reply:
<img width="816" alt="Screen Shot 2022-02-25 at 10 27 05 PM" src="https://user-images.githubusercontent.com/2701446/155734200-d3d4a8f0-023f-471f-bf51-e5a90006fd26.png">

it'll complain about generic type inference

Reply:
got it.